examples/ha-nat-gateway/main.tf

```diff
@@ -69,22 +71,25 @@ module "nat-zone-2" {
 
 module "nat-zone-3" {
   source     = "../../"
+  name       = "${var.network_name}-"
   region     = "${var.region}"
   zone       = "${var.zone3}"
   network    = "${google_compute_subnetwork.default.name}"
   subnetwork = "${google_compute_subnetwork.default.name}"
 }
 
 module "mig1" {
-  source             = "github.com/GoogleCloudPlatform/terraform-google-managed-instance-group"
+  source             = "GoogleCloudPlatform/managed-instance-group/google"
+  version            = "1.1.13"
   region             = "${var.region}"
   zone               = "${var.zone1}"
-  name               = "ha-nat-mig"
+  name               = "${var.network_name}-mig"
   size               = 2
   access_config      = []
-  target_tags        = ["nat-${var.region}"]
+  target_tags        = ["${module.nat-zone-1.routing_tag_regional}"]
```

Comment:
will this change our current tags?

Reply:
Nope. That's from an example implementation file. We don't use it.